docs/reference/transform/transforms-at-scale.asciidoc

```diff
@@ -0,0 +1,197 @@
+[role="xpack"]
+[[transform-scale]]
+= Working with {transforms} at scale
+++++
+<titleabbrev>{transforms-cap} at scale</titleabbrev>
+++++
+
+{transforms-cap} convert existing {es} indices into summarized indices, which 
+provide opportunities for new insights and analytics. The search and index 
+operations performed by {transforms} use standard Elasticsearch features so 
+similar considerations for working with Elasticsearch at scale are often 
+applicable to transforms. If you experience performance issues, start by 
+identifying the bottleneck areas (search, indexing, processing, or storage) then 
+review the relevant considerations in this guide to improve performance. It also 
+helps to understand how {transforms} work as different considerations apply 
+depending on whether or not your transform is running in continuous mode or in 
+batch.
+
+In this guide, you’ll learn how to:
+
+* Understand the impact of configuration options on the performance of 
+  {transforms}.
+
+**Prerequisites:**
+
+These guildelines assume you’re already familiar with: 
```

Comment:
> These guildelines assume you’re already familiar with: 

From subsequent paragraphs, it seems like the assumption is that you're not just familiar with these concepts, but you have a transform that you're now attempting to tune/debug.  Should that be the prerequisite instead?